cli/utils.py

```diff
@@ -76,7 +82,6 @@ def dict_merge(first, second, path=None,
                 conflict_resolver(first, second, key)
         else:
             first[key] = second[key]
-    return first
```

Comment:
Please take a look on the comment a gave on the return value in 'string_to_list' method by mistake.